exec-healthz/Makefile

```diff
@@ -1,17 +1,69 @@
-all: push
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Hmm, well ok we need to do the same for contrib as well. I suppose this could be the first.